lib/dotcom_web/live/system_status.ex

```diff
@@ -49,4 +97,154 @@ defmodule DotcomWeb.Live.SystemStatus do
     </details>
     """
   end
+
+  defp alerts_examples() do
```

Comment:
[This comment](https://github.com/mbta/dotcom/pull/2353/files#r1946663221) applies to this as well.